samples/cpp/stitching_detailed.cpp

```diff
@@ -813,7 +817,7 @@ int main(int argc, char* argv[])
             }
             blender->prepare(corners, sizes);
         }
-        else if (!timelapser)
+        else if (!timelapser && timelapse)
```

Comment:
I see, this should work. However, can this if-else combo be taken out of the image loop? Looks like it is independent of images and is executed only once.